src/components/custom-field-input-multiple-choice/custom-field-input-multiple-choice.jsx

```diff
@@ -85,93 +93,95 @@ function CustomFieldInputMultipleChoice(props) {
   }, [expanded]);
 
   return (
-    <FormControl
-      error={props.helpText}
-      label={props.label}
-      labelId={`${props.id}-label`}
-      id={`${props.id}-autocomple`}
-      onKeyDown={onKeyDown}
-      readOnly={props.readOnly}
-    >
-      <div
-        className={classContainer}
-        onClick={onClick}
-        role="presentation"
+    <div ref={wrapperRef}>
```

Comment:
It could be interesting to bake this into `FormControl` instead of adding another `<div>`. But this is non-blocking. I think it might cause further merge conflicts with the other tracks.

Reply:
I will hold off on that refactor for now then.